pkg/operator/k8sutil/deployment.go

```diff
@@ -48,7 +48,7 @@ func GetDeploymentSpecImage(clientset kubernetes.Interface, d apps.Deployment, c
 func WaitForDeploymentImage(clientset kubernetes.Interface, namespace, label, container string, initContainer bool, desiredImage string) error {
```

Comment:
I was going to suggest we make this method smarter to wait indefinitely if the pod is still in container starting state. But if this is only called by the integration tests, seems fine to leave it as is, and maybe move it to the test code base instead of having it in the k8sutil package.

Reply:
This was a quick-and-dirty test to see if it would help the integrations pass. If it does, I was thinking we could possibly add a parameter to the function to change the wait time.